CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+# [0.7.0](https://github.com/nervosnetwork/ckb-sdk-ruby/tree/rc/v0.7.0) (2019-03-22)
+
+
+### Features
+
+* add basic wallet ([2e14b49](https://github.com/nervosnetwork/ckb-sdk-ruby/commit/2e14b49))
+* add rpc interface ([a4b1f47](https://github.com/nervosnetwork/ckb-sdk-ruby/commit/a4b1f47))
+* init project ([1cb5623](https://github.com/nervosnetwork/ckb-sdk-ruby/commit/1cb5623))
```

Comment:
Please add an entry explaining that this sdk is extracted from ruby demo.

Reply:
In changelog or readme ?

Reply:
changelog

Reply:
Done